api/app/signals/apps/services/domain/filescanner.py

```diff
@@ -0,0 +1,44 @@
+# SPDX-License-Identifier: MPL-2.0
+# Copyright (C) 2021 Gemeente Amsterdam
+"""
+Check file types, both for fresh uploads and stored files.
+"""
+import mimetypes
+
+import magic
+
+ALLOWED_MIME_TYPES = ['image/jpeg', 'image/png', 'image/gif']
+
+
+class BadFileError(Exception):
+    pass
+
+
+class UploadScannerService:
```

Comment:
If we call this the `FileScannerService` it can be used in more cases. For example if we want to scan files in a job instead of when uploaded.

Reply:
For this PR I will not add a more general `FileScannerService`, instead I added a ticket to our backlog to add a `FileScannerService` (in JIRA, which is not public). Ticket number is 3739.